### PR TITLE
Add a one-page User Guide

### PR DESCRIPTION
**Pull request by wandernauta**
This user guide would help users to get started with the program.

### CODE REVIEW

File: Report/Chapters/Manual.tex [hunk 1]
```diff
@@ -1,2 +1,46 @@
...
+
+\textbf{Squares!}
+First, we're building a program that calculates the product of two numbers.
+To do that, right-click (or tap-and-hold) anywhere to open the \emph{function menu}, if it isn't open already.
+(It's the big, purple box: it's hard to miss.)
+The function menu has a number of buttons to choose from.
+For now, click the button that says \emph{Display Block} to add a display to your program.
+The display block doesn't do much on its own: it only displays the value that it's connected to.
+Next, add a \emph{value block} by clicking the respective button on the function menu.
+You'll be asked to provide a value: for now, we're going to try a floating point value: enter 1234.0.
```
Comment (jjkester) [inline Report/Chapters/Manual.tex]: Maybe clarify what to enter by wrapping it with ```\texttt```.

File: Report/Chapters/Manual.tex [hunk 2]
```diff
@@ -1,2 +1,46 @@
...
+You should now have two blocks: drag the output block so that it's a bit below the value block.
+Then, from the \emph{Numeric types} category, click \emph{(*)}, the multiplication function.
```
Comment (jjkester) [inline Report/Chapters/Manual.tex]: Personally I like the function name in \texttt instead of \emph.

File: Report/Chapters/Manual.tex (hunk 1 above)
Comment (wandernauta) [inline Report/Chapters/Manual.tex]: Good idea, will do.

File: Report/Chapters/Manual.tex [hunk 3]
```diff
@@ -1,2 +1,46 @@
...
+The multiplication function has two inputs.
+Connect both inputs (the top dots) to the value block you created by clicking and dragging from one dots to the other.
+Then attach the function block's output anchor (the bottom dot) to the display block's input.
+The square of the number you picked (1522756.0) should now appear in the output block.
```
Comment (jjkester) [inline Report/Chapters/Manual.tex]: The value could be in \texttt

File: Report/Chapters/Manual.tex [hunk 4]
```diff
@@ -1,2 +1,46 @@
...
+First of all, grab a \texttt{sin} function from the Numeric category.
+The sine function normally takes a floating point value and returns a floating point value, not a function like the graph block needs.
+To have it return a function, drag the \textit{knot}, the dot between the input type and the output type, to the left.
+Our function block now takes no inputs and provides in a function.
+Feed that function into the graph block and watch your sine wave appear.
+
+\textbf{More!}
+There are more blocks: the \emph{definition block} allows you to turn parts of your program into reusable functions, and the \emph{RGB block} takes red, green and blue floating point values and shows them as a color. 
+The possibilities are endless.
+Have fun!
```
Comment (jjkester) [inline Report/Chapters/Manual.tex]: Maybe a new line before "Have fun!"

File: Report/Chapters/Manual.tex (hunk 3 above)
Comment (wandernauta) [inline Report/Chapters/Manual.tex]: Agreed.

File: Report/Chapters/Manual.tex (hunk 4 above)
Comment (wandernauta) [inline Report/Chapters/Manual.tex]: I agree that it would be better, but it doesn't seem to fit on one page if I do...

**Comment by wandernauta** [comment]
Merged. Thanks for taking a look, @jjkester!

**Pull request merged**